src/FullscreenQuad.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "Drawable.h"
+#include <VertexBufferDX11.h>
+#include <IndexBufferDX11.h>
+
+class FullscreenQuad
```

Comment:
FYI: This class has a draw() function, but it's not a drawable on purpose. If you want to draw it, you need to call draw() explicitly.